linux/reproducibility/ae_reproducibility_verifier/README.md

```diff
@@ -1,100 +1,62 @@
-Verify Intel(R) Prebuilt AE Reproducibility:
-================================
----------
+# How to Verify Intel(R) Prebuilt AE Reproducibility
 
 ## Purpose
----
-Since Intel(R) prebuilt AEs have been signed by Intel's private key, it's hard to verify AE reproduciblity by comparing your local AE builds with Intel AEs using "diff" command.  
-So this doc will give an approach using your local AE builds to verify Intel(R) prebuilt AEs are:
 
-* Built using same code and toolsets that being opensourced
-* Signed using same config.xml that being opensourced
+Since Intel(R) prebuilt AEs have been signed by Intel's private key, it's not easy to verify your AE builds against Intel(R) Prebuilt AEs in a common way (e.g. comparing the file checksum).  
+So the doc gives an approach to guarantee Intel(R) prebuilt AEs are:
```

Comment:
This document outlines a process to guarantee that Intel(R) prebuilt AEs are:

Reply:
Opened https://github.com/intel/linux-sgx/pull/992 to address